packages/catalog-search/src/tests/SearchSuggestions.test.jsx

```diff
@@ -45,6 +61,20 @@ describe('<SeachSuggestions />', () => {
     expect(screen.getByText('View all results')).not.toBeNull();
   });
 
+  test('renders no errors when no authoring orgs found for programs data', () => {
+    renderWithRouter(<SearchSuggestions
```

Comment:
verified this tesst actually does fail the same way as reported inthe ticket, in the absence of the fix :)